src/dashboard/src/settings/common.py

```diff
@@ -35,6 +35,8 @@
     'gearman_server': {'section': 'Dashboard', 'option': 'gearman_server', 'type': 'string'},
     'shibboleth_authentication': {'section': 'Dashboard', 'option': 'shibboleth_authentication', 'type': 'boolean'},
     'copy_files_timeout': {'section': 'Dashboard', 'option': 'copy_files_timeout', 'type': 'int'},
+    'version': {'section': 'Dashboard', 'option': 'version', 'type': 'string'},
+    'agent_code': {'section': 'Dashboard', 'option': 'agent_code', 'type': 'string'},
```

Comment:
Do we still need this or the edits in `CONFIG_DEFAULTS`?

Reply:
No... again, will remove.